themes/pyladiestheme/templates/base.html

```diff
@@ -48,6 +48,7 @@
             <!-- <li><a data-scroll href="/#resources">Materiały</a></li> -->
             <li><a data-scroll href="/#coc">Kodeks</a></li>
             <li><a data-scroll href="/#partners">Partnerzy</a></li>
+            <li><a href="/#blog">Blog</a></li>
```

Comment:
Isn't the class `data-scroll` needed here? What is it needed for then? ( I really don't know :P )